test/functional/dbcrash.py

```diff
@@ -99,14 +87,16 @@ def restart_node(self, node_index, expected_tip):
         # and make sure that recovery happens.
         raise AssertionError("Unable to successfully restart node %d in allotted time", node_index)
 
-    # Try submitting a block to the given node.
-    # Catch any exceptions that indicate the node has crashed.
-    # Returns true if the block was submitted successfully; false otherwise.
     def submit_block_catch_error(self, node_index, block):
+        """Try submitting a block to the given node.
+
+        Catch any exceptions that indicate the node has crashed.
+        Returns true if the block was submitted successfully; false otherwise."""
+
         try:
             self.nodes[node_index].submitblock(block)
             return True
-        except (httplib.CannotSendRequest, httplib.RemoteDisconnected) as e:
+        except (http.client.CannotSendRequest, http.client.RemoteDisconnected) as e:
```

Comment:
I think this RemoteDisconnected exception may not exist pre-3.5, which would explain this travis failure:
https://travis-ci.org/bitcoin/bitcoin/jobs/248398016#L3321

I'm not sure what the best way to handle that is...